ftests/nuxeo-platform-explorer-ftests/pom.xml

```diff
@@ -8,54 +8,24 @@
     <version>11.2-SNAPSHOT</version>
   </parent>
 
-  <artifactId>nuxeo-platform-explorer-ftests</artifactId>
-  <name>Marketplace Platform Explorer - Webdriver Functionnal Tests</name>
+  <artifactId>nuxeo-explorer-ftests-parent</artifactId>
+  <name>Explorer - Functional Tests Parent</name>
+  <packaging>pom</packaging>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.nuxeo.packages</groupId>
-      <artifactId>nuxeo-platform-explorer-package</artifactId>
-      <type>zip</type>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
+  <modules>
+    <module>explorer-ftests-base</module>
+    <module>explorer-ftests-nomode</module>
+    <module>explorer-ftests-sitemode</module>
+  </modules>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.nuxeo.build</groupId>
-        <artifactId>ant-assembly-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>start-tomcat</id>
-            <phase>pre-integration-test</phase>
-            <goals>
-              <goal>build</goal>
-            </goals>
-            <configuration>
-              <targets>
-                <target>prepare-packages</target>
-                <target>prepare-environment</target>
-                <target>start</target>
-              </targets>
-            </configuration>
-          </execution>
-          <execution>
-            <id>stop-tomcat</id>
-            <phase>post-integration-test</phase>
-            <goals>
-              <goal>build</goal>
-            </goals>
-            <configuration>
-              <targets>
-                <target>stop</target>
-                <target>cleanup-environment</target>
-              </targets>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.nuxeo.ftests</groupId>
+        <artifactId>nuxeo-explorer-ftests-base</artifactId>
+        <version>${nuxeo.platform.version}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
```

Comment:
Following the [comment on nuxeo-jsf-ui](https://github.com/nuxeo/nuxeo-jsf-ui/pull/68#discussion_r442105374), this block should go to the nuxeo root pom.